projects/components/src/datagrid/datagrid.component.ts

```diff
@@ -323,7 +330,26 @@ export class DatagridComponent<R> implements OnInit, AfterViewInit, AfterViewChe
     /**
      * Desired height of the grid. If unspecificed, the grid fills the parent container.
      */
-    @Input() height: number;
+    @Input() set height(height: number) {
+        this._height = height;
+        if (this.height) {
+            this.node.nativeElement
+                .querySelector('clr-datagrid')
+                .style.setProperty('--datagrid-height', this.height + 'px');
+        } else {
+            this.node.nativeElement.querySelector('clr-datagrid').style.setProperty('--datagrid-height', 'unset');
+        }
```

Comment:
This is probably nicer with a variable for what the height should be
```suggestion
        const height = this.height ? `${this.height}px` : 'unset';
        this.node.nativeElement.querySelector('clr-datagrid').style.setProperty('--datagrid-height', height);
```

Reply:
And instead of using `querySelector`, can't we just create a reference to the underlying datagrid element instead? We should avoid using `querySelector`